nri/css/style.css

```
@@ -0,0 +1,367 @@
+@charset "UTF-8";
+
+/* 共通部分
+-------------------- */
+
+html {
+    font-size: 100%;
+    font-feature-settings: "palt"
+}
+body{
```

コード整形（vscodeの機能などを利用）しましょう

```suggestion
body {
```